### ISSUE TITLE
State of the project? Docs discrepancies. Version 5.4.5 vs 6.0.0

### ISSUE THREAD

**Issue by cristian-eriomenco** [kidjp85/react-id-swiper#479]
I've noticed a lot of discrepancies between the docs and sandboxes.

The react-id-swiper doesn't seem to work properly (coverflow, nav buttons) with any version of swiper past 5.4.5. Yet the docs state:

`npm install react-id-swiper@latest swiper@latest`
and
`import 'swiper/css/swiper.css';` - this file doesn't exist 
instead it should be :
`import "swiper/swiper-bundle.min.css";`

There are some PRs which are sitting there for a while now.

Swiper itself seems to have bundled react support in itself: https://swiperjs.com/react/

**Long story short is the project abandoned?**

**Comment by claudiopedrom** [comment]
Same doubt here.

**Comment by bbaki** [comment]
Seems abandoned to me. I'm not quite sure what was the purpose of this project anyway. Seems like a react wrapper for swiper but as you have pointed out, swiper seems to support react out of the box so is there any other reason for this project?
Woud be cool however to have an official info on this.

**Comment by ItsTarik** [comment]
Agree, very odd .. i think this packages is helping with `swiper` custom builds, Swiper alone has some issues for example using it along with nextjs the dev build or on resizing the screen causes some odd behaviors, I'm trying to use this packages so maybe those issues will disappear but i agree, since swiper.js is already exporting react primitives already there is no sense having this package also `swiper.js 7.0.0` is changing its core implementation and i found it also confusing that is supporting just `esm` modules !!
That's very confusing for the community, is there someone to give a clear explanation about all these stories ?
